packagingbuild/bionic/Dockerfile

```diff
@@ -47,17 +47,26 @@ RUN apt-get update && \
         devscripts debhelper dh-make libldap2-dev libsasl2-dev && apt-get clean
 
 # Install fresh pip and co
-RUN curl https://bootstrap.pypa.io/get-pip.py | python3.6 - virtualenv==16.6.0 pip==20.0.2 wheel setuptools cryptography; \
+RUN curl https://bootstrap.pypa.io/get-pip.py | python3.6 - virtualenv==20.4.0 pip==20.3.3 wheel setuptools cryptography; \
       pip3.6 install --upgrade requests[security] && rm -rf /root/.cache
 
-# We use our dh-virtualenv version, since it fixes shebangd lines rewrites
+# This line just busts Docker's cache so it re-runs the next line
+# The GitHub API will return different results when the branch HEAD changes
+# See https://stackoverflow.com/a/39278224
+# ADD https://api.github.com/repos/StackStorm/dh-virtualenv/git/refs/heads/stackstorm_patched version.json
+# RUN rm version.json
+#
+# We use our dh-virtualenv version, since it fixes shebangd lines rewrites.
+# The shebangd rewrites are in master branch, but that requires dependency
+# on debhelper 12.
```

Comment:
I think in the future it's OK to switch to debhelper `12`.

From what I see we're on `10` right now: https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/compat

Reply:
When I tried on bionic in our images, the latest debhelper it could download was 11. I don't think 12 is in the bionic repos.

Reply:
Makes sense, good to know!
So probably `10` for Ubuntu Xenial then.